roaring/roaring.go

```diff
@@ -519,18 +519,40 @@ func (b *Bitmap) Optimize() {
 }
 
 //hoping this in-lines
```

Comment:
It doesn't seem like this comment applies anymore. Does it make sense to remove it?